code/controllers/subsystem/timer.dm

```diff
@@ -411,10 +410,8 @@ SUBSYSTEM_DEF(timer)
 		prev.next = next
 		next.prev = prev
 	else
-		if(prev)
-			prev.next = null
-		if(next)
-			next.prev = null
+		prev?.next = null
```

Comment:
Woooooaaaah. `?.`? Awesome.

Reply:
Yeah, they basically do what you see above, only they condense it to a single line.

Reply:
That's glorious. Another awesome operator.